src/server.rs

```diff
@@ -124,9 +130,30 @@ impl<A> Server<A> {
         &mut self.acceptor
     }
 
-    /// Returns a mutable reference to the Http builder.
-    pub fn http_builder(&mut self) -> &mut Builder<TokioExecutor> {
-        &mut self.builder
```

Comment:
I'd like to see this API remain available, as I use it to configure many settings for both HTTP/1 and HTTP/2 :eyes:

Reply:
The idea here is to expose anything `Builder` exposes through `axum-server`s interface.
What method exactly are you unable to access with the new API?

Reply:
Hmm, I don't see how I would access these:
```rust
serve
    .http_builder()
    .http1()
    .timer(hyper_util::rt::TokioTimer::new())
    .keep_alive(true)
    .header_read_timeout(Duration::from_secs(5))
    .max_buf_size(32768)
    .http2()
    .timer(hyper_util::rt::TokioTimer::new())
    .keep_alive_interval(Duration::from_secs(16))
    .keep_alive_timeout(Duration::from_secs(5))
    .max_header_list_size(512 * 1024)
    .max_send_buf_size(64 * 1024)
    .max_concurrent_streams(16);
```

Reply:
Ah yes, I assume if we expose `Builder::http1()` and `Builder::http2()` this would cover all the methods you have listed?
The PR doesn't include them

Reply:
I have an intermediate solution that does not breaks compatibility that can be used, see #156. Let me know what you think.

Reply:
Glad it doesn't break compatibility. Still don't personally understand the API or the use-case but you have to convince the maintainer, not me ;)